test/policy.js

```diff
@@ -63,6 +63,15 @@ function() {
   assert(!result);
 });
 
+it('fails with another disallowed labels',
+function() {
+  var testPolicy = { labels: { disallow: ['com.swipely.iam-docker.msi-explicit-identity']}};
+  var testContainer = { Config: { Labels: { 'com.swipely.iam-docker.msi-explicit-identity': 12345}}};
```

Comment:
Should we test `AGENT_FILL` here too?

Reply:
we could, policy is always fluid. I was just sanity checking myself.